docs/src/adr/020adr-prioritization.md

```diff
@@ -0,0 +1,204 @@
+taken towards the work:
+
+| Category        | Approach                                           |
+|-----------------|----------------------------------------------------|
+| Quick wins      | Do ASAP                                            |
+| Major projects  | Sustain long term focus                            |
+| Fill-ins        | Do in spare time, but never taking time from above |
+| Thankless tasks | Avoid, unless there's literally nothing else to do |
+
+#### Scoring
+
+We determine which category a work item falls under based on the _impact_ and
+_effort_ scores assigned to the work. We therefore need a shared understanding
+of how to assess the impact and effort of a work item. We register only 3 levels
+on each axis to keep the cognitive load of reckoning scores minimal.
+
+Scores are recorded by affixing labels to the github issue tracking the work item.
+
+##### Effort
+
+Effort is scored best on a rough estimate of the amount of focused time it would
+take to complete a work item:
+
+- Easy **`e0`**: Can be completed within about 1 day
+- Medium **`e1`**: Can be completed within 3 days
+- Difficult **`e2`**: Will take 5 or more days
+
+Ideally, as soon as we recognized that an actively planned ticket will take more
+than 5 days of focused work, we would factor it into smaller tickets, allowing
+us to avoid perpetually prolonged monster tickets. But some people may prefer to
+track "major projects" with a single issue rather than a milestone, and this
+doesn't pose a significant problem.
+
+##### Impact
+
+Impact is scored based on considering impact in the follow 4 domains:
+
+1. _User / customer_: The benefit of the results of a work item to our users and
+   customers
+2. _Mission_: Furtherance of our organization's mission and objectives
+3. _Invention_: Advancement of state of the art in formal verification and
+   specification
+4. _Development_: Improvements to our development capacities and bandwidth
+   (which supports advancing the other three factors).
+
+- High **`i0`**:
+  - User / customer: Unblocks critical work
+  - Mission: Advances critical organizational objectives
+  - Invention: Opens up novel verification and specifications abilities
+  - Development: Saves > 3 hours per week
+- Medium **`i1`**:
+  - User / customer: Unblocks non-critical work
+  - Mission: Advances non-critical organizational objectives
+  - Invention: Makes incremental improvement to verification and specification
+    abilities
+  - Development: Saves between 1 to 3 hours per week
+- Low **`i2`**:
+  - User / customer: Improves functionality, but an easy workaround exists
+  - Mission: No significant advance of organizational objectives
+  - Invention: No significant improvement to verification and specification abilities
+  - Development: Saves < 1 hours per week
+
+### Prioritization and evaluation
+
+The initial priority of a requirement should be established at the time work is
```

Comment:
Who is responsible for scoping a piece of work? The person proposing it? The person being asked to do it? It's important to keep in mind that for many tasks, it might not be clear who is working on the task until it is chosen as a next task by someone, so tasks need to be scoped before that happens.

Reply:
Both are responsible.